src/App.css

```diff
@@ -82,14 +82,14 @@
 }
 
 #menu-wrap .dots > div:after {
-  content: '';
+  content: "";
```

Comment:
btw we prefer single quotes
I think you would need to adjust some linter configs.